presto-main/src/main/java/io/prestosql/execution/SqlQueryExecution.java

```diff
@@ -335,6 +336,21 @@ public Duration getTotalCpuTime()
         return scheduler.getTotalCpuTime();
     }
 
+    @Override
+    public DataSize getScannedBytes()
+    {
+        SqlQueryScheduler scheduler = queryScheduler.get();
+        Optional<QueryInfo> finalQueryInfo = stateMachine.getFinalQueryInfo();
+        if (finalQueryInfo.isPresent()) {
+            return finalQueryInfo.get().getQueryStats().getRawInputDataSize();
+        }
+        if (scheduler == null) {
+            return DataSize.succinctBytes(0);
+        }
+        StageInfo root = scheduler.getStageInfo();
+        return new DataSize(stateMachine.getScanedBytes(root), BYTE);
```

Comment:
Since [`DataSize(double size, Unit unit)`](https://github.com/airlift/units/blob/master/src/main/java/io/airlift/units/DataSize.java#L107) construtor is deprecated, we can use `DataSize.ofBytes(long)` instead.